### PR TITLE
Add H2 intermittent failures to known issues

### PR DESCRIPTION
**Pull request by mhajas**

### CODE REVIEW

File: release_notes/topics/21_0_0.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+This release includes the following known issues:
+
+* After upgrading H2 database from the version `1.4.199` to the version
+`1.4.200`, we encountered intermittent failures when the configured isolation
+level is lower than `REPEATABLE_READ`. In Keycloak distribution, the
+`REPEATABLE_READ` isolation level is used by default when H2 is configured
+as the database. Refer to the following
+https://github.com/keycloak/keycloak/issues/14917#issuecomment-1311763163[comment]
+for more details.
```
Comment (hmlnarik) [inline release_notes/topics/21_0_0.adoc:13]: Add: This does not change support level for H2 which is still only packed for development purposes and not supported in production.

**Review by andymunro — APPROVED** [review]
Looks good, @mhajas

**Review by stianst — COMMENTED** [review]
If we believe this is important enough to mention should we perhaps add it to the [DB guide](https://www.keycloak.org/server/db) instead?

**Comment by ahus1** [comment]
@mhajas - the documentation has been move to the main repository. If this is still relevant, add a new PR / issue to the main Keycloak repository.

**Pull request closed without merging**